<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>本地视频文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_videoPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_OpenVideo">
        <property name="text">
         <string>打开视频频</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CCustomOpenGLWidget" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>780</width>
        <height>460</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>70</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_play">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cutImage">
        <property name="text">
         <string>截图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_barrage">
        <property name="text">
         <string>弹幕</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCustomOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">ccustomopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
